runtime/vm/ContinuationHelpers.cpp

```diff
@@ -238,11 +243,6 @@ yieldContinuation(J9VMThread *currentThread)
 	 * must be maintained for weakly ordered CPUs, to unsure that once the continuation is again available for GC scan (on potentially remote CPUs), all CPUs see up-to-date stack .
```

Comment:
Update slightly how the above comment starts:
We don't need atomic to reset the state (reset carrier ID), since....

Also update the order text the same:

swap-stacks
writeBarrier
state initial (reset carrier ID)